python/python3-build/python3-build.SlackBuild

```diff
@@ -22,16 +23,18 @@
 #  OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
 #  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+# 20230507 46and2: New maintainer, updated version and deps.
+#   -build process changed
+
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=python3-build
-VERSION=${VERSION:-0.7.0}
+SRCNAM=$(echo $PRGNAM | sed "s/python3-//")
```

Comment:
Hi!
Since the shell used is forcibly bash, we may use bashisms here.
As a pure bash equivalent to calling echo and sed in a subshell, I'd suggest this line :
SRCNAM=${PRGNAM#python3-*}